File: src/main/java/org/pcha/benthos/language/psi/BloblangTokenType.java
package org.pcha.benthos.language.psi;

import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class BloblangTokenType extends com.intellij.psi.tree.IElementType {
    public BloblangTokenType(@NotNull @NonNls String debugName) {
        super(debugName, org.pcha.benthos.language.BloblangLanguage.INSTANCE);
    }

    public String toString() {
        return "SimpleTokenType." + super.toString();
    }
}

File: src/main/kotlin/com/github/pcha/intellij/bloblang/psi/BloblangTokenType.kt
package com.github.pcha.intellij.bloblang.psi

import com.github.pcha.intellij.bloblang.BloblangLanguage
import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls

class BloblangTokenType(debugName: @NonNls String) :
    IElementType(debugName, BloblangLanguage.INSTANCE) {
    override fun toString(): String {
        return "BloblangTokenType." + super.toString()
    }
}
